packages/frontend/app/register/weekly-meetings/components/get-weekly-meetings.tsx

```diff
@@ -56,39 +60,48 @@ export function GetWeeklyMeetings() {
 
     // Fetch all local meetings at this location
     dispatch(
-      fetchLocalMeetingsThunk({
+      fetchMeetingsThunk({
         latitude: String(state.userCoords.latitude),
         longitude: String(state.userCoords.longitude),
       }),
     );
   }, [state.userCoords, dispatch]);
 
+  const renderLocationSearch = () => {
+    if (!useCurrentLocation && !state.displayCongregations.length)
+      return <LocationSearch setUseCurrentLocation={setUseCurrentLocation} />;
```

Comment:
Potentially why CI is breaking. Add a default returning <></> or something

Reply:
CI is actually failing because of my tests so not too sure what's going on there...

Reply:
Oh sorry my bad I thought the frontend build was failing. Ignore this